scripts/ci/install_azdev.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
This file should be deleted as well.

Reply:
Looks like this file is not referenced anywhere.